src/locale/misc/views/en.yml

```diff
@@ -9,6 +9,10 @@ columnDialog:
       description: First and last name of person
       keywords: name combo
       title: First and Last Name
+    localPerson:
+      columnTitle: Assignee
```

Comment:
Ooh, already a situation where having separate `title` and `columnTitle` paid off. 💯